### PR TITLE
Add links to flutter and toit.io.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

**Review by HelenaMeyer — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
-# Toit Flutter Demo
+# Flutter Demo
```
Comment (erikcorry) [inline README.md:1]: Suggest:
Demo of controlling IoT devices with Flutter (using Toit.io).

File: README.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 This project is intended to be used as starting point for Flutter
-applications that want to communicate with the Toit server. Just
-pick the functionality that is most similar to what you need, and
-remove the remaining pages.
+applications that want to communicate with embedded devices or the
+Toit server. Just pick the functionality that is most similar to what
+you need, and remove the remaining pages.
 
 ## Implementation Note
 The Toit APIs need access to the Internet.
```
Comment (erikcorry) [inline README.md:17]: I don't understand this sentence.

**Review by erikcorry — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (floitsch) [reply README.md]: done. without the "(using Toit.io)".

File: README.md (hunk 2 above)
Comment (floitsch) [reply README.md]: changed.

**Pull request merged**